api-connection.html

```diff
@@ -100,7 +100,203 @@ <h2 id="function_newConnection">new Connection(config)</h2>
+          </ul>
+          <li><code>azure-active-directory-access-token</code></li>
+{% highlight javascript %}
+"authentication": {
+  ...,
+  "options": {
+    "token": value;
+  }
+}         
+{% endhighlight %}
+          <ul>
+            <li>token (string): A user need to provide `token` which they retrieved elsewhere to forming the connection.</li>
+            <p><h4>How to get the token</h4></p>
+            <ul>
+              <h4>Things that need to considered before using the fallowing scripts:</h4>
+              <ul>
+                <ol>
+                    <li>
+                      Two PowerShell scripts need to be run within a msi-vm or a msi-app-service environment,since it need to pick up the specific preset environment variables from the system or reach a specific msi end point.
+                    </li>
+                    <li>
+                      An azure managed identity(user or system) must be enabled first then register to an Azure AD authentication-enabled data source. Otherwise, if you have registered an identity into for example a database and the system identity is diabled from azure for an Azure VM, you will get a connection error due to that the data source will not recognize the identity.For more detials on this, this is a <a href="https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/tutorial-windows-vm-access-sql" target="_blank"><i><b>link</b></i></a> from microsoft. 
+                    </li>
+                    <li>
+                      Tedious supports an authentication type: azure-active-directory-password. By using this authentication, tedious will handle the process for retrieving the azure token internally, so the user does not have to retrieve the azure token by themselves using the following provoded tips. The "azure-active-directory-password" should be the first choice since that Tedious will handle the token retrieve process for the users.
+                    </li>
+                </ol>
+              </ul>
+              <h4>A PowerShell script for retrieving the token within a msi-app-service:</h4>
+              Here is a <a href="https://docs.microsoft.com/en-us/azure/app-service/overview-managed-identity?tabs=dotnet" target="_blank"><i><b>link</b></i></a> for microsoft side doc for retrive the token.
+{% highlight powershell %}
+# The version of the token API to be used. Please use "2019-08-01" or later 
+# (unless using Linux Consumption, which currently only offers "2017-09-01").
+$apiVersion = "2019-08-01"
+# The Azure AD resource URI of the resource for which a token should be obtained.
+# This could be one of the provide URIs within this link 
+# [https://docs.microsoft.com/en-us/azure/active-directory/
+# managed-identities-azure-resources/
+# services-support-managed-identities#
+# azure-services-that-support-azure-ad-authentication]
+# or any other resource URI.
```

Comment:
```suggestion
<# The Azure AD resource URI of the resource for which a token should be obtained.
This could be one of the provide URIs within this link 
[https://docs.microsoft.com/en-us/azure/active-directory/
managed-identities-azure-resources/
services-support-managed-identities#
azure-services-that-support-azure-ad-authentication]
or any other resource URI. #>
```
Makes URL copy and paste-able in the documentation